src/store/base.ts

```diff
@@ -0,0 +1,47 @@
+// BASE
+export class BaseStore<S extends {}> {
```

Comment:
seems not used anymore - shall we remove it?

Reply:
Yes right.